### DISCUSSION TITLE
ToolTip/Popup opens on top of all windows

### DISCUSSION THREAD

**Discussion by ILD3225** [AvaloniaUI/Avalonia#15212] [Q&A] [answered]
### Describe the bug

ToolTip/Popup opens on top of all windows.
![image](https://github.com/AvaloniaUI/Avalonia/assets/14194323/d3afc927-fddf-4527-a047-1c96a6001136)

### To Reproduce

Run the "ControlCatalog.NetCore" project from the "Samples" folder.
Open the "ToolTip" section, turn on ToggleSwitch.
Move any window to the tooltip area.

### Expected behavior

ToolTip should be visible at the level of its window, and not appear on other windows

### Avalonia version

11.0.10

### OS

Windows

### Additional context

As a solution, you need to move this code above
![image](https://github.com/AvaloniaUI/Avalonia/assets/14194323/08241d49-b3a4-44a9-8382-0da2e02f7075)

**Comment by maxkatz6** [comment, marked as answer]
It's not a bug. That's by design and that's how tooltips/popups normally work in any GUI framework.

If you want to disable it for you app, you can update BuildAvaloniaApp configuration like so:
```diff
    .UsePlatformDetect()
+    .With(new Win32PlatformOptions { OverlayPopups = true })
+    .With(new X11PlatformOptions { OverlayPopups = true })
+    .With(new AvaloniaNativePlatformOptions { OverlayPopups = true })
```
per each desktop platform

**Reply by ILD3225** [reply]
Thank you

**Comment by ZSYMAX** [comment]
I have a new problem when  set 'OverlayPopups =true',the ToolTip Popup not close when pointer exit the control,Is that a feature or bug?
![tooltip](https://github.com/user-attachments/assets/5035f341-8a81-4c46-99b6-ddcb89e03f21)
@maxkatz6
